x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -276,5 +315,342 @@ export function getDetailsItems(anomaly: AnomaliesTableRecord, filter: EntityCel
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyCharacteristicsTooltip',
+            {
+              defaultMessage: 'Anomaly characteristics impact blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyCharacteristics"
+              defaultMessage="Anomaly characteristics impact"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: <ImpactVisual score={explanation.anomaly_characteristics_impact} />,
+    });
+  }
+
+  if (explanation.single_bucket_impact !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.singleBucketTooltip',
+            {
+              defaultMessage: 'Single bucket impact blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.singleBucket"
+              defaultMessage="Single bucket impact"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: <ImpactVisual score={explanation.single_bucket_impact} />,
+    });
+  }
+  if (explanation.multi_bucket_impact !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.multiBucketTooltip',
+            {
+              defaultMessage: 'Multi bucket impact blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.multiBucket"
+              defaultMessage="Multi bucket impact"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: <ImpactVisual score={explanation.multi_bucket_impact} />,
+    });
+  }
+  if (explanation.high_variance_penalty !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.highVarianceTooltip',
+            {
+              defaultMessage: 'High variance interval blah blah blah',
```

Comment:
Description text needed

Reply:
@szabosteve and @valeriy42 can you please help me with this tooltip text.

Reply:
Updated in https://github.com/elastic/kibana/pull/142999/commits/45313cddd3db35b4d6d83f8b238aa9367e9953fa

Reply:
The long and simplified answer is: We have detected a periodic pattern in the time series. The current bucket is in the part of the periodic interval where data can vary a lot. This leads to a large confidence interval since we have seen data all over the place in this part of the period before. Hence, we reduce the anomaly score for the current anomaly in this region. 

Now, how to fit it in a tooltip?